backend/api/Areas/Keycloak/Models/AgencyModel.cs

```diff
@@ -0,0 +1,60 @@
+using System;
```

Comment:
Is this just to get around automapper or is there another reason this exists? it seems odd to me that this class exists but has no relationship to the existing AgencyModel.

Reply:
While duplication of models isn't great, always remember that models are specific to their endpoint(s).  It's one of the reasons some design implementations don't use models and only use dynamic objects (in the effort to remove boilerplate hard to keep up-to-date models).

The models in the `/Areas/Keycloak/Models` are for the `Keycloak` controller and are or maybe tightly coupled with the logic related to integrating Keycloak and PIMS.